source/_posts/2022-12-20-year-of-voice.markdown

```diff
@@ -44,7 +44,7 @@ With Home Assistant we want to make a privacy and locally focused smart home ava
 
 With Home Assistant we prefer to get the things we’re building in the user's hands as early as possible. Even basic functionality allows users to find things that work and don’t work, allowing us to address the direction if needed.
 
-A voice assistant has a lot of different parts: hot word detection, speech to text, intent recognition, intent execution, text-to-speech. Making each work in every language is a lot of work. The most important part is the intent recognition and intent execution. We need to be able to understand your commands and execute them.
+A voice assistant has a lot of different parts: hot word detection, speech-to-text, intent recognition, intent execution, text to speech. Making each work in every language is a lot of work. The most important part is the intent recognition and intent execution. We need to be able to understand your commands and execute them.
```

Comment:
I don't think we should have touched `text-to-speech`. A rebase went wrong?

Reply:
quite possibly... Thank you for the catch!

Reply:
quite possibly... Thank you for the catch! #27613